bitnami/postgresql/requirements.yaml

```diff
@@ -1,4 +1,4 @@
 dependencies:
   - name: common
     version: 0.x.x
-    repository: https://charts.bitnami.com/bitnami
\ No newline at end of file
+    repository: https://charts.bitnami.com/bitnami
```

Comment:
The requirements.lock should be updated, shouldn't it?

Reply:
done